Gemfile

```diff
@@ -62,11 +62,11 @@ end
 group :test do
   gem 'database_cleaner'
   gem 'json-schema'
+  gem 'mocha'
   gem 'shoulda-matchers'
   gem 'simplecov', require: false
+  gem 'timecop'
```

Comment:
Not used yet, but figured it would be useful since we deal with dates and time a lot